API/Controllers/UserController.cs

```diff
@@ -37,6 +37,15 @@ public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userE
             return Ok();
         }
 
+        [HttpPost("user-edit-info")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userEditProfileInfo, string email)
```

Comment:
I think you should add "Async" at the end of the name of the asynchronous method